src/main/java/uk/gov/register/providers/DownloadExceptionMapper.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.register.providers;
+
+import uk.gov.register.exceptions.DownloadException;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class DownloadExceptionMapper implements ExceptionMapper<DownloadException> {
+    @Override
+    public Response toResponse(final DownloadException exception) {
```

Comment:
I might steal one of Michaela's comments from another PR here and ask *can we use the ExceptionView to display this error?*